apps/docs/content/components/LogoSuite/index.mdx

```diff
@@ -22,7 +21,7 @@ Use the logo suite component to showcase a list of logos, such as for clients, s
 
 A logo suite is intended to build credibility, trust, and provide social proof by displaying consistent logos.
 
-Logo suite should be used for groups of logos more than 4 and less than 12. In scenarios hoping to showcase more than 12 logos, considering stacking two components together. To create the best visual consistency and balance, rearrange and resize the logos when necessary to create similar area distribution and attention to each of the logos.
+Logo suite should be used for groups of logos more than 4 and less than 12. To create the best visual consistency and balance, rearrange and resize the logos when necessary to create similar area distribution and attention to each of the logos.
 
 ### Arrangement
 
```

Comment:
Unable to add these comments as these sections haven't changed, but will need to...

1. Can we remove section for `Contextual information` as it still recommends adding a heading a description
  ![Screenshot 2024-08-16 at 15 15 28](https://github.com/user-attachments/assets/de5fe697-d757-4226-8bf5-bd92df9283d1)
2. Remove 'default muted', unless we are dropping new `default` variant and sticking with muted..
  ![Screenshot 2024-08-16 at 15 30 27](https://github.com/user-attachments/assets/f538844d-6951-436f-81ed-093b3dfbdd5b)
3. Could we please update this example to use the description instead of Heading now. This will help us a) match the marquee example further above it, but also reinforce that we shouldn't use LogoSuite.Heading arbitrarily
  ![Screenshot 2024-08-16 at 15 32 26](https://github.com/user-attachments/assets/49aacec0-163d-411a-acf3-1bf783f26328)

Reply:
4. In the `React` tab, the "Default" example still includes a heading and description but these are now optional. Can we remove those in the "Default" and move this example to appear immediately  after and call it "Optional heading and description". This should make it super obvious what the minimum expectations are, and match [what we do in other components](https://primer-90ea61f842-26139705.drafts.github.io/brand/components/BreakoutBanner/react#optional-description).

![Screenshot 2024-08-16 at 15 35 26](https://github.com/user-attachments/assets/ac18a65c-eda7-4031-8bcf-6088fc9146ca)

Reply:
All updated (along with a small copy tweak too) in b6616fc962b20c019bfb709a89b452f1f8f4ddd9

For further context, Dani and I agreed that the docs images will be updated in a subsequent PR once he's updated the LogoSuite component in Figma

Reply:
In that last suggestion (4) I've updated it but left the heading in the Default example and just made it `visuallyHidden` if that works for you?